torch/testing/_internal/common_device_type.py

```diff
@@ -3,9 +3,9 @@
 import inspect
 import runpy
 import threading
-import typing
-from typing import Union, List, TypeVar, Tuple, Generic, ClassVar, Iterable, Any, Optional
 from functools import wraps
+# how to introduce ClassVar without importing typing?
```

Comment:
could you explain why can't you do `from typing import ClassVar` what exception were you getting?

Reply:
As we previous said "no need to import typing and from typing import xxx", I just wonder whether we need it..